yang/frr-vrf.yang

```diff
@@ -0,0 +1,61 @@
+module frr-vrf {
+  yang-version 1.1;
+  namespace "http://frrouting.org/yang/vrf";
+  prefix frr-vrf;
+
+  organization
+    "Free Range Routing";
+  contact
+    "FRR Users List:       <mailto:frog@lists.frrouting.org>
+     FRR Development List: <mailto:dev@lists.frrouting.org>";
+  description
+    "This module defines a model for managing FRR VRF.";
+
+  revision 2019-12-06 {
+    description
+      "Initial revision.";
+  }
+
+  feature netns;
+
+  container lib {
+    list vrf {
+      key "name";
+      description
+        "VRF.";
+      leaf name {
+        type string {
+          length "1..16";
+        }
+        description
+          "VRF name.";
+      }
+
+      leaf id {
+        type uint32 {
+          range "1..4294967295";
+        }
+        description
+          "VRF Id.";
+      }
+
+      container netns_vrf {
+        if-feature "netns";
+        leaf ns_name {
+          type string {
+            length "1..16";
```

Comment:
The netns can be a full filesystem path. We probably don't need any `length` restriction here.

Reply:
Removed the `length` restriction from netns `ns_name`.